hooks/useArtists.tsx

```diff
@@ -7,6 +7,7 @@ import useArtistMode from "./useArtistMode";
 import saveArtist from "@/lib/saveArtist";
 import useInitialArtists from "./useInitialArtists";
 import useCreateArtists from "./useCreateArtists";
+import useYouTubeChannel from "@/hooks/useYouTubeChannel";
```

Comment:
why are you importing this hook in useArtists?
- It's only being used in the artist popup right?